Bottom: fd11353a0070122b80ba0234bba7e4308759ccc5
Top:    6f801eb7eafa041946dd433917a5aff6825ca50a
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-24 16:30:47 -0500

Remove libicu52 from Dockerfile

Newest upstream versions of php:5.6-apache are based on debian stretch and 
use libicu57, which is already installed. Besides this, libicu-dev would 
already install libicuXX.


---
diff --git a/docker/docker-mediawiki/Dockerfile b/docker/docker-mediawiki/Dockerfile
index 6aaefd61..54421165 100644
--- a/docker/docker-mediawiki/Dockerfile
+++ b/docker/docker-mediawiki/Dockerfile
@@ -13,7 +13,6 @@ RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get install --yes --no-install-recommends \
         git \
         imagemagick \
-        libicu52 \
         libicu-dev \
         libldap2-dev \
     && \
